contracts/testing_utils/DummyWETHToken.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This isn't required as well

Reply:
This is also require for testing.

Reply:
There are several contract libraries that you can use for deploying a dummy token, no? Anyway as long its just for testing, this is fine.